yrs/src/lib.rs

```diff
@@ -86,3 +89,31 @@ pub use crate::types::xml::XmlTextRef;
 pub use crate::types::GetString;
 pub use crate::types::Observable;
 pub use crate::update::Update;
+use rand::RngCore;
+
+pub type Uuid = std::sync::Arc<str>;
+
+pub fn uuid_v4<R: RngCore>(rng: &mut R) -> Uuid {
+    let mut b = [0u8; 16];
+    rng.fill_bytes(&mut b);
+    let uuid = format!(
+        "{:x}{:x}{:x}{:x}-{:x}{:x}-{:x}{:x}-{:x}{:x}-{:x}{:x}{:x}{:x}{:x}{:x}",
```

Comment:
I've fixed it, however I don't want to add it to `lib0` - the reason is that it puts a dependency on a `rand` crate, which now puts us in a problematic spot (we have its version pinned, since in the latest versions they dropped support for web assembly).